docs/topics/about-tools.adoc

```diff
@@ -11,4 +12,5 @@ The {ProductName} ({ProductShortName}) provides several tools to assist you in t
 * Web console
 * {PluginName} for Eclipse and Red Hat CodeReady Studio
 * {ProductShortName} extension for Visual Studio Code
+* {ProductShortName} plugin for IntelliJ IDEA
 * {MavenName}
```

Comment:
This is a case where I think the attribute looks a bit silly. "Maven plugin" would make more sense and it's a lot easier to remember.